common/v2/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -346,7 +352,10 @@ export const TxReceiptUI = ({
             <AssetIcon uuid={asset.uuid} size={'24px'} />
             <Amount
               assetValue={`${parseFloat(assetAmount()).toFixed(6)} ${assetTicker()}`}
-              fiatValue={`$${convertToFiat(parseFloat(assetAmount()), assetRate()).toFixed(2)}
+              fiatValue={`${Fiats[settings.fiatCurrency].symbol}${convertToFiat(
```

Comment:
Can we do a helper function to find the fiat instead this kind of lookup?

(Same for other files)

Reply:
sure sure. that'd be good to help with `fiatCurrency of undefined` errs too :)